llama : rename batch_all to batch

danbev opened the pull request:
This commit addresses the TODO in the code to rename the `batch_all` parameter to `batch` in `llama_decode_internal`.



- [x] I have read the [contributing guidelines](https://github.com/ggerganov/llama.cpp/blob/master/CONTRIBUTING.md)
- Self-reported review complexity:
  - [x] Low
  - [ ] Medium
  - [ ] High

danbev commented on src/llama.cpp:
```diff
@@ -14447,10 +14447,10 @@ static void llama_graph_compute(
 
     lctx.is_encoding = false;
-    const uint32_t n_tokens_all = batch_all.n_tokens;
+    const uint32_t n_tokens_all = batch.n_tokens;
```
Would it make sense to also rename `n_tokens_all` to `n_tokens` after this change?

ggerganov replied:
It will clash with the `n_tokens` defined a bit further down:

```c++
const uint32_t n_tokens = ubatch.n_tokens;
```

compilade commented:
Note that this partly conflicts with #8526, although I'm not against the rename.

danbev commented:
> Note that this partly conflicts with #8526, although I'm not against the rename.

Ah I was not aware of that. I'll close this PR in that case. Thanks.

danbev committed:
llama : rename batch_all to batch

This commit addresses the TODO in the code to rename the `batch_all`
parameter to `batch` in `llama_decode_internal`.

danbev commented:
Reopened as it looks like this change did not make into https://github.com/ggerganov/llama.cpp/pull/8526.
